packages/core/directives/map.ts

```diff
@@ -446,7 +463,7 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
   private _setCenter() {
     let newCenter = {
       lat: this.latitude,
-      lng: this.longitude,
+      lng: this.longitude
```

Comment:
trailing commas are recommended now 🤷

Reply:
Not critical in my opinion, but Section 5.2.1 of the Google JavaScript Style Guide states that the trailing comma should be included.

Reply:
definitely not critical, but code reviews are meant to be pedantic 😉